polling-app-server/src/main/java/com/example/polls/model/User.java

```diff
@@ -109,4 +116,20 @@ public void setPassword(String password) {
     public void setRoles(Set<Role> roles) {
         this.roles = roles;
     }
+
+    public boolean isActive() {
```

Comment:
so fun with getters,you think this is short,i.....

Reply:
I don't understand does not one need getters setters, the author used it, if not Lombok could be used.